evm/spec/exceptions.tex

```diff
@@ -0,0 +1,44 @@
+\section{Exceptions}
+\label{exceptions}
+
+Sometimes, when executing user code (i.e. contract or transaction code), the EVM halts exceptionally (i.e. outside of a STOP, a RETURN or a REVERT).
+When this happens, the CPU table invokes a special instruction with a dedicated operation flag \texttt{exception}.
+Exceptions can only happen in user mode; triggering an exception in kernel mode would make the proof unverifiable.
+No matter the exception, the handling is the same:
+
+-- The opcode which will trigger the exception is not executed. The operation flag set is \texttt{exception} instead of the opcode's flag.
```

Comment:
```suggestion
-- The opcode which would trigger the exception is not executed. The operation flag set is \texttt{exception} instead of the opcode's flag.
```
or:
```suggestion
-- The opcode which triggers the exception is not executed. The operation flag set is \texttt{exception} instead of the opcode's flag.
```